core/ir/opnd_api.h

```diff
@@ -3266,18 +3266,13 @@ DR_API
 /**
  * Assumes that \p reg is a DR_REG_ 32-bit register constant.
  * Returns the string name for \p reg.
+ * \note It uses the global dcontext_t to determine the ISA mode.  If the ISA mode is a
+ * synthetic one (e.g., #DR_ISA_REGDEPS), it returns the name of a DR_REG_V virtual
```

Comment:
It may be worth replacing all the `DR_REG_V` with `#DR_REG_V0, etc.` or something so we have a real link: here and in all Doxygen comments added by this PR.

Reply:
Done.
Turns out that that was the only doxygen comment (we removed the other public API).